docs/cookbook/adding-new-attribute-to-an-entity.md

```diff
@@ -44,25 +44,6 @@ class Product extends BaseProduct
         parent::__construct($productData, $variants);
     }
 
-    /**
-     * @param \Shopsys\ShopBundle\Model\Product\ProductData $productData
-     * @return \Shopsys\ShopBundle\Model\Product\Product
-     */
-    public static function create(BaseProductData $productData)
-    {
-        return new self($productData, null);
-    }
-
-    /**
-     * @param \Shopsys\ShopBundle\Model\Product\ProductData $productData
-     * @param \Shopsys\ShopBundle\Model\Product\Product[] $variants
-     * @return \Shopsys\ShopBundle\Model\Product\Product
-     */
-    public static function createMainVariant(BaseProductData $productData, array $variants)
-    {
-        return new self($productData, $variants);
-    }
-
```

Comment:
```
new self
```
was changed into 
```
new static
```

and now it is not needed to override it since the block got no modifications and static will do the thing like it is in this example http://sandbox.onlinephpfunctions.com/code/d8fd08427751e308052234c66ff2f56f44445c4f

Reply:
Ok, I believe you :)